frontend/packages/data-portal/app/components/Viewer/ViewerPage.tsx

```diff
@@ -141,12 +162,15 @@ export function ViewerPage({
   const iframeRef = useRef<NeuroglancerAwareIframe>(null)
   const hashReady = useRef<boolean>(false)
   const helpMenuRef = useRef<MenuDropdownRef>(null)
+  const voxelSpacing = useRef<number>(0)
+  const alignmentId = useRef<number>(0)
 
   const shareSnackbar = useAutoHideSnackbar()
   const snapSnackbar = useAutoHideSnackbar()
 
   const depositionConfigs = buildDepositionsConfig(run.annotations)
   const shouldShowAnnotationDropdown = Object.keys(depositionConfigs).length > 0
+  const shouldShowTomogramDropdown = tomograms.length > 1
```

Comment:
We can actually probably remove this check, the tomogram menu has helpful info even if only one tomogram. To confirm